example/src/App.tsx

```diff
@@ -172,64 +172,64 @@ const App = () => {
         AppLovinMAX.loadRewardedAd(REWARDED_AD_UNIT_ID);
       }, retryDelay * 1000);
     });
-    AppLovinMAX.addEventListener('OnRewardedAdClickedEvent', (adInfo) => {
+    AppLovinMAX.addRewardedAdClickedEventListener((adInfo) => {
       setStatusText('Rewarded ad clicked');
     });
-    AppLovinMAX.addEventListener('OnRewardedAdDisplayedEvent', (adInfo) => {
+    AppLovinMAX.addRewardedAdDisplayedEventListener((adInfo) => {
       setStatusText('Rewarded ad displayed');
     });
-    AppLovinMAX.addEventListener('OnRewardedAdFailedToDisplayEvent', (adInfo) => {
+    AppLovinMAX.addRewardedAdFailedToDisplayEventListener((adInfo) => {
       setRewardedAdLoadState(adLoadState.notLoaded);
       setStatusText('Rewarded ad failed to display');
     });
-    AppLovinMAX.addEventListener('OnRewardedAdHiddenEvent', (adInfo) => {
+    AppLovinMAX.addRewardedAdHiddenEventListener((adInfo) => {
       setRewardedAdLoadState(adLoadState.notLoaded);
       setStatusText('Rewarded ad hidden');
     });
-    AppLovinMAX.addEventListener('OnRewardedAdReceivedRewardEvent', (adInfo) => {
+    AppLovinMAX.addRewardedAdReceivedRewardEventListener((adInfo) => {
       setStatusText('Rewarded ad granted reward');
     });
-    AppLovinMAX.addEventListener('OnRewardedAdRevenuePaid', (adInfo) => {
+    AppLovinMAX.addRewardedAdRevenuePaidListener((adInfo) => {
       setStatusText('Rewarded ad revenue paid: ' + adInfo.revenue);
     });
 
     // Banner Ad Listeners
-    AppLovinMAX.addEventListener('OnBannerAdLoadedEvent', (adInfo) => {
+    AppLovinMAX.addBannerAdLoadedEventListener((adInfo) => {
       setStatusText('Banner ad loaded from ' + adInfo.networkName);
     });
-    AppLovinMAX.addEventListener('OnBannerAdLoadFailedEvent', (errorInfo) => {
+    AppLovinMAX.addBannerAdLoadFailedEventListener((errorInfo) => {
       setStatusText('Banner ad failed to load with error code ' + errorInfo.code + ' and message: ' + errorInfo.message);
     });
-    AppLovinMAX.addEventListener('OnBannerAdClickedEvent', (adInfo) => {
+    AppLovinMAX.addBannerAdClickedEventListener((adInfo) => {
```

Comment:
I'd make sure the docs [ticket](https://app.asana.com/0/337926977135969/1203814034303571) is up-to-date.

Reply:
Thanks for finding `addEventListener('OnBannerAdClickedEvent'`.